dask_saturn/core.py

```diff
@@ -68,6 +68,12 @@ class SaturnCluster(SpecCluster):
     """
 
     # pylint: disable=unused-argument,super-init-not-called,too-many-instance-attributes
+    def __await__(self):
```

Comment:
It's probably just this and then put the original pylint disable string before `__init__` as well.

```suggestion
    # pylint: disable=unused-argument
    def __await__(self):
```